tests/observability/metrics/test_general_metrics.py

```diff
@@ -94,9 +94,9 @@ def test_vm_name_in_virt_launcher_label(self, fedora_vm_without_name_in_label):
 
 class TestVirtHCOSingleStackIpv6:
```

Comment:
Please mark this as ipv6 and ensure we have a test job that runs it against ipv6 clusters too. Otherwise, you are never checking that this metrics is generating non-zero value ever.

Reply:
I still don't see ipv6 marker on this test.